.github/workflows/pr_build.yaml

```diff
@@ -12,10 +13,36 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: 1.19.3
+      - name: Load cached deps
+        uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
```

Comment:
I don't see any step in here to populate this cache. I'm not convinced we need it yet either. The number of dependencies in spiffe-csi is quite small.

Reply:
OK, removed the commit